test/functional/eval/system_spec.lua

```diff
@@ -354,6 +360,8 @@ describe('systemlist()', function()
       eq(5, eval('v:shell_error'))
       eval([[systemlist('this-should-not-exist')]])
       eq(1, eval('v:shell_error'))
+      eval([[systemlist('"ping" "-n" "1" "127.0.0.1"')]])
+      eq(0, eval('v:shell_error'))
```

Comment:
Is this to test #7698? Doesn't it add 1 second to each test?

Reply:
No. The first request runs immediately. `-n 1` guarantees 1 request only because, by default, it sends up to 5 requests.

Reply:
Ok, still it would be clearer if we could instead check the output of something like echo or cat.

    eq(..., [[systemlist('"echo" "foo" "bar" "baz"')]])